addons/mail/views/mail_activity_views.xml

```diff
@@ -123,7 +123,7 @@
                         </group>
                         <group>
                             <field name="date_deadline"/>
-                            <field name="user_id"/>
+                            <field name="user_id" context="{'internal_only': True}"/>
```

Comment:
Why not using a domain on the field ?

Reply:
We don't want the domain to be applied whenever there is a search called on the `res.users`. We need it for one specific view.